pyproject.toml

```diff
@@ -70,8 +71,8 @@ odxtools = "odxtools.__init__:_main"
 [tool.setuptools]
 packages = [ "odxtools" ]
 
-[tool.setuptools.dynamic]
-version = {attr = "odxtools.version.__version__"}
+[tool.setuptools_scm]
+write_to = "odxtools/version.py"
```

Comment:
write_to is deprecated, use version_file instead

Reply:
in this case, the [documentation](https://pypi.org/project/setuptools-scm/) is outdated as well. also, when using `version_file=` pip produces
```
TypeError: Configuration.__init__() got an unexpected keyword argument 'version_file'
```

Reply:
version_file was added in setuptools_scm v8, what version are you using?

See https://github.com/pypa/setuptools_scm/blob/main/CHANGELOG.rst

Reply:
Nevermind v8 of setuptools_scm has not been published yet